webui/react/src/services/api.ts

```diff
@@ -20,11 +20,11 @@ import { decodeExperimentList, encodeExperimentState } from './decoder';
 
 export { isAuthFailure, isLoginFailure, isNotFound } from './utils';
 
-const address = serverAddress();
+const ApiConfig : Api.Configuration = { basePath: serverAddress() };
```

Comment:
**question:** I'm not sure where the changes are on the swagger end that causes this change. I like this much better than having to pass in an `undefined` argument, but just not sure what resulted in this usage change.

Reply:
nothing this is a no-op. I had a TODO here that I removed. but basically once, and if, we start to use the intended way of authentication with the generated code we'd need this configuration file.